src/JsonEditorWidget.php

```diff
@@ -125,12 +154,76 @@ public function init()
         }
 
         parent::init();
+
+        if ($this->registerCKEditorAsset) {
+            CKEditorAsset::register($this->getView());
+        }
+
+        if ($this->registerJoditAsset) {
+            JoditAsset::register($this->getView());
+
+            // jodit options: https://xdsoft.net/jodit/docs/options.html
+            $joditConfig = '{
+              "buttons": ["classSpan", "bold", "italic", "paragraph", "ol", "ul", "link", "image", "preview", "source"],
+              "toolbarAdaptive": false,
+              "hidePoweredByJodithidePoweredByJodit": true,
+              "spellcheckspellcheck": false,
+              "controls": {
+                "classSpan": {
+                  "list": {}
+                }
+              }
+            }';
+
+            $json = Yii::$app->settings->getOrSet('jodit.config', $joditConfig, 'jsoneditor', 'object');
```

Comment:
Settings should be optional - see: https://github.com/dmstr/yii2-json-editor/pull/13